contracts/whitelists/whitelist-immutable-flex/src/helpers.rs

```diff
@@ -0,0 +1,45 @@
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+use cosmwasm_std::{to_json_binary, Addr, QuerierWrapper, QueryRequest, StdResult, WasmQuery};
+
+use crate::{
+    msg::{ConfigResponse, QueryMsg},
+    state::Config,
+};
+
+/// CwTemplateContract is a wrapper around Addr that provides a lot of helpers
+/// for working with this.
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
+pub struct WhitelistImmutableFlexContract(pub Addr);
+
+impl WhitelistImmutableFlexContract {
+    pub fn addr(&self) -> Addr {
+        self.0.clone()
+    }
+
+    pub fn includes(&self, querier: &QuerierWrapper, address: String) -> StdResult<bool> {
+        let includes: bool = querier.query(&QueryRequest::Wasm(WasmQuery::Smart {
+            contract_addr: self.addr().into(),
+            msg: to_json_binary(&QueryMsg::HasMember { address })?,
+        }))?;
+        Ok(includes)
+    }
+
+    pub fn address_count(&self, querier: &QuerierWrapper) -> StdResult<u64> {
```

Comment:
Is there address count or is this mint count for address? I think the second name makes more sense?

Reply:
There is a separate Member query that returns the mint count for each address. AddressCount is legacy and returns the total number of members.